pootle/core/initdb.py

```diff
@@ -36,6 +41,34 @@ def initdb():
     create_default_languages()
 
 
+def _create_object(model_klass, **criteria):
+    instance, created = model_klass.objects.get_or_create(**criteria)
+    if created:
+        logger.debug(
+            "Created %s: '%s'"
+            % (instance.__class__.__name__, instance))
+    else:
+        logger.debug(
+            "%s exists - skipping: '%s'"
+            % (instance.__class__.__name__, instance))
+    return instance
+
+
+def _create_pootle_user(**criteria):
+    user = _create_object(get_user_model(), **criteria)
+    user.set_unusable_password()
+    user.save()
+    return user
+
+
+def _create_pootle_permission_set(permissions, **criteria):
+    permission_set = _create_object(PermissionSet, **criteria)
+    if permission_set.positive_permissions != permissions:
```

Comment:
Actually just noticed that we want to check if all the specified `permissions` are on `permission_set.positive_permissions` and add the missing ones (while keeping any other extra permission there), while here we are checking that `permission_set.positive_permissions` is exactly `permissions`. Also blindly setting it to `permissions` is not an option.